Indicators/PercentagePriceOscillator.cs

```diff
@@ -41,7 +41,7 @@ public PercentagePriceOscillator(string name, int fastPeriod, int slowPeriod, Mo
         /// <param name="slowPeriod">The slow moving average period</param>
         /// <param name="movingAverageType">The type of moving average to use</param>
         public PercentagePriceOscillator(int fastPeriod, int slowPeriod, MovingAverageType movingAverageType = MovingAverageType.Simple)
-            : this(string.Format("PPO({0},{1})", fastPeriod, slowPeriod), fastPeriod, slowPeriod, movingAverageType)
+            : this($"PPO({fastPeriod},{slowPeriod})", fastPeriod, slowPeriod, movingAverageType)
```

Comment:
The PPO indicator was already supporting `IIndicatorWarmUpPeriodProvider` as it derives indirectly from MACD, right?